docker.local/config/sc.yaml

```diff
@@ -151,10 +154,10 @@ smart_contracts:
     block_reward:
       block_reward: 1
       qualifying_stake: 1
-      sharder_ratio: 80
-      miner_ratio: 20
-      blobber_capacity_ratio: 20
-      blobber_usage_ratio: 80
+      sharder_ratio: 10
```

Comment:
+1 accidentally added here

Reply:
No, actually that one is actually part of the PR. Change the block reward rations.